### ISSUE TITLE
Step 4: setup of Alpine errors out on 'ifup: too few parameters for line "iface"'

### ISSUE THREAD

**Issue by biemster** [biemster/rpi4_alpine_netboot#1]
The first command in step 4, `setup-ntp`, gives an error `ifup: too few parameters for line "iface"`. This causes subsequent steps to fail as well, because the clock will be wrong. Manually running `$ rc-service networking restart` gives the same error, indicating a serious networking issue.

**Comment by biemster** [comment]
running setup-alpine instead of the /sbin/setup-* scripts individually initially fixed this issue, but installing packages further that require listening on a socket gives this error again. So something is clearly off in networking.

**Comment by biemster** [comment]
local loopback interface `lo` seems to be missing, could that be it?

**Comment by biemster** [comment]
It seems to originate in the initialisation of eth0, in /etc/network/interfaces the second interface is not specified:
```
auto lo
iface lo inet loopback

auto 
iface  inet dhcp
        hostname localhost
```

**Comment by biemster** [comment]
fixed in 09cc044.
